src/Controller/TaskController.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+namespace App\Controller;
+
+use App\Entity\Task;
+use App\Form\TaskType;
+use App\Repository\TaskRepository;
+use Doctrine\ORM\EntityManagerInterface;
+use Symfony\Bundle\FrameworkBundle\Controller\AbstractController;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\Routing\Attribute\Route;
+
+class TaskController extends AbstractController
+{
+
+    public function __construct(private EntityManagerInterface $em)
```

Comment:
```suggestion
    public function __construct(private readonly EntityManagerInterface $em)
```
Je crois que tu peux mettre ça